packages/core/config/environment.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(/* environment, appConfig */) {
     navi: {
       user: 'navi_user',
       dataEpoch: '2013-01-01',
-      dataSources: [{ name: 'dummy', uri: 'https://data.naviapp.io' }],
+      dataSources: [{ name: 'dummy', uri: 'https://data.naviapp.io', type: 'bard-facts' }],
```

Comment:
Should we call `bard-facts` `fili`?

Reply:
We should, there is an open issue to rename everything bard. #573